File: URI/1 - INICIANTE/Python/1866 - Conta.py
z = int(input())
while z > 0:
    num = int(input())
    temp = s = 0
    for y in range(0 , num):
        if temp == 0:
            s += 1
            temp = 1
        else:
            s -= 1
            temp = 0
    print(s)
    z -= 1

File: URI/1 - INICIANTE/C/1866 - Conta.c
#include <stdio.h>

int main(){
	int z, num, temp, s, y;
	scanf(" %d", &z);
	while(z > 0){
		s = 0;
		temp = 0;
		scanf(" %d", &num);
		for(y = 0; y < num; y++){
			if(temp == 0){
				s += 1;
				temp = 1;
			}else{
				s -= 1;
				temp = 0;
			}
		}
		printf("%d\n", s);
		z--;
	}
	return 0;
}
